code/datums/components/quality.dm

```diff
@@ -6,10 +6,13 @@
 #define FINE_QUALITY 1.1
 #define SUPERIOR_QUALITY 1.2
 #define EXCEPTIONAL_QUALITY 1.3
-#define MASTERFUL_QUALITY 1.5
-#define ARTIFACT_QUALITY 1.75 // big upgrade coz it is so rare
+#define MASTERFUL_QUALITY 1.4
+#define ARTIFACT_QUALITY 1.4
```

Comment:
why ARTIFACT_QUALITY = MASTERFUL_QUALITY  ?

Reply:
ill be adding some special stuff to the artifacfs

Reply:
Great